.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,32 @@
+name: Run Link Checker
+on: [push]
+
+jobs:
+  initialize:
+    name: Clone Theme
+    runs-on: ubuntu-latest
+    steps:
+    - name: Initialize Theme
+      run: |
+        sudo apt-get update && sudo apt-get install -y git
+        git init
+        git submodule update --init --recursive --depth 1
+  build:
+    name: Build Website
+    runs-on: ubuntu-latest
+    steps:
+    - name: Build
+      env:
+        HUGO_VERSION: 0.55.6
+      run: |
+        sudo apt-get update && sudo apt-get install -y curl \
+                                                           git \
+                                                           openssh-client \
```

Comment:
Is the OpenSSH client needed here?

Reply:
I take it based on the dockerfile since seems like it can't use custom dockerimage.